dearpygui/demo.py

```diff
@@ -1867,15 +1904,54 @@ def create_table_set(policy):
                                 # series belong to a y axis
                                 dpg.add_line_series(sindatax, sindatay, label="0.5 + 0.5 * sin(x)")
 
-                    with dpg.tree_node(label="Stair Series"):
-
-                        with dpg.plot(label="Stair Plot", height=400, width=-1):
+                    with dpg.tree_node(label="Filled Line Series"):
+                        dpg.add_checkbox(label="fill", tag="fill_series_cb", default_value=False, 
+                                         callback=lambda: dpg.configure_item("filled_line_series", shaded=dpg.get_value("fill_series_cb")))
+                        dpg.add_checkbox(label="segment", tag="segment_series_cb", default_value=False, 
+                                         callback=lambda: dpg.configure_item("filled_line_series", segments=dpg.get_value("segment_series_cb")))
+                        with dpg.plot(label="Filled Line Plot", height=400, width=-1):
                             dpg.add_plot_legend()
                             dpg.add_plot_axis(dpg.mvXAxis, label="x")
                             with dpg.plot_axis(dpg.mvYAxis, label="y"):
-                                dpg.add_stair_series(sindatax, sindatay, label="0.5 + 0.5 * sin(x)")
-
+                                dpg.add_line_series(sindatax, sindatay, tag="filled_line_series", label="0.5 + 0.5 * sin(x)", )
```

Comment:
`sin(x)", )`:

That's an interesting comma :). Looks as if more changes were to happen here but somehow haven't happened... Is it intentional?
A similar construct can also be found in 7 (!) other places throughout this file.

Reply:
No actually... I didn't even notice I put this changes, sorry! :)